lib/libs/email/content/respondToRai/emailTemplates/MedSpaCMS.tsx

```diff
@@ -7,31 +7,27 @@ import {
 } from "../../email-components";
 import { BaseEmailTemplate } from "../../email-templates";
 
-export const MedSpaCMSEmail = (props: {
+export const MedSpaCMSEmail = ({
+  variables,
+}: {
   variables: Events["RespondToRai"] & CommonEmailVariables;
-}) => {
-  const variables = props.variables;
-  const previewText = `Medicaid SPA ${variables.id} RAI Response Submitted`;
-  const heading = "The OneMAC Submission Portal received a Medicaid SPA RAI Response Submission:";
-
-  return (
-    <BaseEmailTemplate
-      previewText={previewText}
-      heading={heading}
-      applicationEndpointUrl={variables.applicationEndpointUrl}
-      footerContent={<BasicFooter />}
-    >
-      <LoginInstructions appEndpointURL={variables.applicationEndpointUrl} />
-      <PackageDetails
-        details={{
-          "State or Territory": variables.territory,
-          Name: variables.submitterName,
-          "Email Address": variables.submitterEmail,
-          "Medicaid SPA Package ID": variables.id,
-          Summary: variables.additionalInformation,
-        }}
-      />
-      <Attachments attachments={variables.attachments} />
-    </BaseEmailTemplate>
-  );
-};
+}) => (
+  <BaseEmailTemplate
+    previewText={`Medicaid SPA ${variables.id} RAI Response Submitted`}
```

Comment:
```suggestion
    previewText={`Medicaid SPA RAI Response for ${variables.id} Submitted`}
```

https://confluenceent.cms.gov/display/MACPRO3/Email+Notifications+for+Package+Actions#EmailNotificationsforPackageActions-CMSUsers.1

Reply:
Resolving with https://github.com/Enterprise-CMCS/macpro-mako/pull/1117/commits/83e2e2471ea17a68a61e2e1cdedbb65ab51bca35